pom.xml

```diff
@@ -534,9 +535,9 @@
                         <!-- Setup links to external api docs
                          -->
                         <links>   
-                            <link>https://javadoc.io/doc/ch.qos.logback/logback-core/${logback.version}</link>
-                            <link>https://javadoc.io/doc/ch.qos.logback/logback-classic/${logback.version}</link>
-                            <link>https://javadoc.io/doc/ch.qos.logback/logback-access/${logback.version}</link>
+                            <link>https://javadoc.io/doc/ch.qos.logback/logback-core/${logback.core.version}</link>
+                            <link>https://javadoc.io/doc/ch.qos.logback/logback-classic/${logback.core.version}</link>
+                            <link>https://javadoc.io/doc/ch.qos.logback/logback-access/${logback.access.version}</link>
```

Comment:
I think the package group artifact id are wrong here. Shouldn't that be:

`<link>https://javadoc.io/doc/ch.qos.logback.access/common/${logback.access.version}</link>`

Reply:
good point. corrected 🙇